check.sh

```diff
@@ -2,11 +2,13 @@
 
 set -ex
 
+ON_GITHUB_CI=false
 EXIT_STATUS=0
 
 # If not running on Github's CI, discard the summaries
 if [ -z "${GITHUB_STEP_SUMMARY+x}" ]; then
     GITHUB_STEP_SUMMARY=/dev/null
+    ON_GITHUB_CI=true
```

Comment:
I think this boolean is inverted

Reply:
It is yeah, the if statement is supposed to detect `$GITHUB_STEP_SUMMARY` not being set.

Reply:
Ah, yes, you are definitely right, I can fix that

Reply:
Fixed in dc76e174